core/chains/evm/client/errors.go

```diff
@@ -160,6 +160,19 @@ var harmony = ClientErrors{
 	Fatal:                   harmonyFatal,
 }
 
+// Klaytn
+// https://github.com/klaytn/klaytn/blob/dev/blockchain/error.go
+// https://github.com/klaytn/klaytn/blob/dev/blockchain/tx_pool.go
+var klaytn = ClientErrors{
+	NonceTooLow:                 regexp.MustCompile(`(: |^)nonce too low$`),
+	TransactionAlreadyInMempool: regexp.MustCompile(`(: |^)(known transaction|there is another tx which has the same nonce in the tx pool)`),
+	TerminallyUnderpriced:       regexp.MustCompile(`(: |^)(transaction underpriced|^intrinsic gas too low)`),
+	LimitReached:                regexp.MustCompile(`(: |^)txpool is full`),
+	InsufficientEth:             regexp.MustCompile(`(: |^)insufficient funds`),
+	TxFeeExceedsCap:             regexp.MustCompile(`(: |^)(invalid gas fee cap|^max fee per gas higher than max priority fee per gas)`),
+	Fatal:                       gethFatal,
```

Comment:
Does klaytn use the exact same error strings as geth here?

Reply:
Mostly. The gethFatal can cover all of Klaytn fatal for chainlink.